src/Lykke.Service.Subscribers/Modules/ServiceModule.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Autofac;
+using AzureStorage.Tables;
+using Common.Log;
+using Lykke.Service.Subscribers.AzureRepositories.Subscribers;
+using Lykke.Service.Subscribers.Core.Repositories;
+using Lykke.Service.Subscribers.Core.Services;
+using Lykke.Service.Subscribers.Core.Settings.ServiceSettings;
+using Lykke.Service.Subscribers.Services;
+using Lykke.SettingsReader;
+
+namespace Lykke.Service.Subscribers.Modules
+{
+    public class ServiceModule : Module
```

Comment:
Can be internal

Reply:
This is a class automatically generated from a template, maybe we should also change the template for services.
https://github.com/LykkeCity/lykke.dotnettemplates/blob/master/Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Modules/ServiceModule.cs